client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/UntracedHotRodOperation.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.client.hotrod.impl.operations;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import net.jcip.annotations.Immutable;
+import org.infinispan.client.hotrod.DataFormat;
+import org.infinispan.client.hotrod.ProtocolVersion;
+import org.infinispan.client.hotrod.configuration.Configuration;
+import org.infinispan.client.hotrod.impl.protocol.Codec;
+import org.infinispan.client.hotrod.impl.transport.netty.ChannelFactory;
+
+/**
+ * An extension of {@link HotRodOperation} that does not include tracing information. We override the
+ * <code>codec</code> with a version that does not support tracing, all operations are executed in the same way.
+ * </p>
+ * This is necessary for backwards compatibility between different versions of the protocol.
+ * @param <T>
+ */
+@Immutable
+public abstract class UntracedHotRodOperation<T> extends HotRodOperation<T> {
```

Comment:
following the above reasoning, I'd rename this class to `NeutralVersionHotRodOperation` or some such